flask_restplus/api.py

```diff
@@ -576,27 +576,34 @@ def handle_error(self, e):
         '''
         got_request_exception.send(current_app._get_current_object(), exception=e)
 
+        include_message_in_error = current_app.config.get("ERROR_INCLUDE_MESSAGE", True)
+        default_data = {}
+
         headers = Headers()
         if e.__class__ in self.error_handlers:
             handler = self.error_handlers[e.__class__]
             result = handler(e)
             default_data, code, headers = unpack(result, HTTPStatus.INTERNAL_SERVER_ERROR)
         elif isinstance(e, HTTPException):
             code = HTTPStatus(e.code)
-            default_data = {
-                'message': getattr(e, 'description', code.phrase)
-            }
+            if include_message_in_error:
```

Comment:
I think code expects every if-branch to create `default_data`. I would do the following.
```
default_data = {}
if include_message_in_error:
    default_data['message'] = getattr(e, 'description', code.phrase)
```

I think you should do the same for the rest. You can also come up with a general function that will handle this behaviour.

Reply:
sorry. I missed L580.